dev/ci/ci-quickchick.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+
+# This script could be included inside other ones
+# Let's avoid to source ci-common twice in this case
+if [ -z "${CI_BUILD_DIR}" ];
```

Comment:
Don't bother with this conditional.

Reply:
Note that this is done already in some files, hence the origin from the cut-and-paste.

Reply:
>Note that this is done already in some files, hence the origin from the cut-and-paste.

That's no reason to keep the copy when flagged in review though.

Reply:
Do you want me to remove the entire conditional altogether or just remove the condition and leave the "then" branch body?